app/vmagent/remotewrite/remotewrite.go

```diff
@@ -492,7 +492,7 @@ func tryPush(at *auth.Token, wr *prompbmarshal.WriteRequest, forceDropSamplesOnF
 		}
 		sortLabelsIfNeeded(tssBlock)
 		tssBlock = limitSeriesCardinality(tssBlock)
-		if sas != nil {
+		if sas.IsEnabled() {
```

Comment:
@hagen1778 don't we need the same for global aggregator?

Reply:
or can we return nil for loadAggregatorsFromData if len(a.as) == nil?

Reply:
Could you please point me to the specific code line or send a follow-up commit?

Reply:
sorry, just seen a variable name in context, expected other variable name for global aggregator